docs/docs/building/rollup-plugin-html.md

```diff
@@ -234,6 +269,10 @@ export interface RollupPluginHTMLOptions {
   transformHtml?: TransformHtmlFunction | TransformHtmlFunction[];
   /** Whether to extract and bundle assets referenced in HTML. Defaults to true. */
   extractAssets?: boolean;
+  /** Define a full absolute url to your site (e.g. https://domain.com) */
+  siteUrl?: string;
```

Comment:
I don't think this is a great name. It should at least be website but not everything is a website. It's rather the deployed origin and base path... is there a common name for that?

Reply:
What about just:

```suggestion
  baseUrl?: string;
```

Reply:
it could be a URL with subfolder as well... e.g. domain.com/my-sub-folder

so it's a full URL to a site/website...

siteUrl or websiteUrl... 
don't really have an idea about a different name 🤔

Reply:
used `absoluteBaseUrl`